alma/alma.php

```diff
@@ -58,9 +58,12 @@ public function __construct()
 
         $this->limited_currencies = ['EUR'];
 
-        $version = explode('.', _PS_VERSION_);
-        $version[3] = (int) $version[3] + 1;
-        $version = implode('.', $version);
+        $version = _PS_VERSION_;
+        if (version_compare(_PS_VERSION_, '1.5.6.1', '<')) {
```

Comment:
pourquoi tu ne réutilises pas la variable $version ?

Reply:
Sorry but `$version` is the variable we want to manipulate and return.
We will modify it only if `_PS_VERSION_` is less than `1.5.6.1` ... so, I prefer using version_compare on `_PS_VERSION_` because it is what we want to test (versus `$version` that is what we want manipulate)

Reply:
I'm agree, I reedit that.